droidlet/lowlevel/hello_robot/hello_robot_mover.py

```diff
@@ -87,12 +89,67 @@ def bot_step(self):
         f = not self.bot.is_moving().value
         return f
 
-    def look_at(self, target, turn_base=True):
+    def relative_pan_tilt(self, dpan, dtilt, turn_base=True):
```

Comment:
follow the same naming conventions throughout of naming variables as `*_rad` if they are in radians?

Reply:
can I get a pass on this for now?  would rather just convert everything to radians once everything is working